build/Build.cmd

```diff
@@ -128,12 +132,38 @@ copy /y Samples\Microsoft.Spark.CSharp\data\* "%SPARKCLR_HOME%\data\"
 popd
 
 @echo Assemble SparkCLR script components
-pushd "%CMDHOME%\..\scripts"
-copy /y *.cmd  "%SPARKCLR_HOME%\scripts\"
-popd
+xcopy /e /y "%CMDHOME%\..\scripts"  "%SPARKCLR_HOME%\scripts\"
 
-@echo zip run directory
+@echo Make distribution
 pushd %~dp0
 if not exist ".\target" (mkdir .\target)
-powershell -f .\zipdir.ps1 -dir "%SPARKCLR_HOME%" -target ".\target\run.zip"
+
+if not defined SPARKCLR_NAME (
+    powershell -f .\localscript\zipdir.ps1 -dir "%SPARKCLR_HOME%" -target ".\target\run.zip"
+    goto :distdone
+)
+
+set TARGET_FILE=.\run\scripts\sparkclr-submit.cmd
+@rem update sparkclr version in sparkclr-submit.cmd file
+powershell -NoProfile -ExecutionPolicy Bypass -Command "((Get-Content %TARGET_FILE%) -replace '\(set SPARKCLR_JAR=.*\)', '(set SPARKCLR_JAR=%SPARKCLR_NAME%.jar)') | Set-Content %TARGET_FILE% -force"
+
+@rem
+@rem Update SparkCLR package version to this release. Example in *.csproj:  
+@rem     <HintPath>..\packages\Microsoft.SparkCLR.1.5.2-SNAPSHOT\lib\net45\CSharpWorker.exe</HintPath>
+@rem
+set TARGET_FILE=..\examples\pi\pi.csproj
+powershell -NoProfile -ExecutionPolicy Bypass -Command "((Get-Content %TARGET_FILE%) -replace '\\Microsoft\.SparkCLR.*\\lib', '\Microsoft.SparkCLR.%ProjectVersion%\lib') | Set-Content -Encoding UTF8 -Path %TARGET_FILE% -force"
+
+@rem
+@rem Update SparkCLR package version to this release. Example in packages.config:  
+@rem     <package id="Microsoft.SparkCLR" version="1.5.2-SNAPSHOT" targetFramework="net45" />
+@rem
+set TARGET_FILE=..\examples\pi\packages.config
```

Comment:
Created a new setversion.ps1 under localmode, so platform devs can update versions at dev time as well. Especially when we fork a new branch, this script should help.